### DISCUSSION TITLE
How do I get route resolved value inside the rules method?

### DISCUSSION THREAD

**Discussion by johnngondi** [spatie/laravel-data#832] [Q&A]
I have this path `[PUT] /steps/{step}`
In the controller, the step is resolved from DB and among the fields, it has the `handle_procurement` flag.

Inside the rules method of the Data class, I have this:

```php
Rule::requiredIf(function() use($context) {
    //I want to check if the step's `handle_procurement` is `true` and return true.
})
```

How can I do that?

I have tried using $this->route('step'), but it is a static method so it ain't working.

**Comment by rubenvanassche** [comment]
You could try: `request()->route('step')`
